files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
```

Comment:
Please can you get in the habit of including full bugzilla URLs in the commit message, as per [team policy](https://etherpad.nue.suse.com/p/cloud-pull-requests)?  In this case it would be https://bugzilla.suse.com/show_bug.cgi?id=974495